lib/generators/sorcery/templates/initializer.rb

```diff
@@ -226,12 +226,20 @@
   # config.line.bot_prompt = "normal"
   # config.line.user_info_mapping = {name: 'displayName'}
 
+  
   # For infromation about Discord API
   # https://discordapp.com/developers/docs/topics/oauth2
   # config.discord.key = "xxxxxx"
   # config.discord.secret = "xxxxxx"
   # config.discord.callback_url = "http://localhost:3000/oauth/callback?provider=discord"
   # config.discord.scope = "email guilds"
+
+  # For infromation about BattleNet API
```

Comment:
Minor typo, `information`

Also it might be good to change `BattleNet` to `Battlenet` so we're consistent with the casing.